faction.cpp

```diff
@@ -409,7 +409,7 @@ void Faction::build_json_report(json& j, Game *game, size_t **citems) {
 	j["administrative"]["show_unit_attitudes"] = (showunitattitudes != 0);
 
 	if(!password_unset) j["administrative"]["password"] = password->const_str();
-	if(Globals->MAX_INACTIVE_TURNS) {
+	if(Globals->MAX_INACTIVE_TURNS != -1) {
```

Comment:
-1 is the signifier for ignore turn inactivity, and if(-1) evaluates to true, so this was a bug but unlikely to get caught since noone actually runs with -1 as the inactive turn setup.